src/blueprints/tv2/timeline-object-factories/tv2-caspar-cg-timeline-object-factory.ts

```diff
@@ -204,21 +207,42 @@ export class Tv2CasparCgTimelineObjectFactory implements Tv2GraphicsElementTimel
     }
   }
 
-  public createSplitScreenLocatorTimelineObject(): CasparCgTemplateTimelineObject {
-    return {} as CasparCgTemplateTimelineObject
-    // TODO: SOF-1695 Should implement this
-    // return {
-    //   id: 'casparCg_locators',
-    //   enable: {
-    //     start: 0
-    //   },
-    //   priority: 1,
-    //   layer: Tv2GraphicsLayer.GRAPHICS_LOCATORS,
-    //   content: {
-    //     deviceType: DeviceType.CASPAR_CG,
-    //     type: CasparCgType.TEMPLATE
-    //   }
-    // }
+  public createSplitScreenLocatorTimelineObject(graphicsSetup: GraphicsSetup, splitScreenConfiguration: SplitScreenConfiguration, locatorLabels?: string[]): CasparCgTemplateTimelineObject {
+    return {
+      id: 'casparCg_locators',
+      enable: {
+        start: 0
+      },
+      priority: 1,
+      layer: Tv2GraphicsLayer.GRAPHICS_LOCATORS,
+      content: {
+        deviceType: DeviceType.CASPAR_CG,
+        type: CasparCgType.TEMPLATE,
+        templateType: 'html',
+        name: `${graphicsSetup.htmlPackageFolder}/${HTML_GRAPHICS_INDEX_FILE}`,
+        useStopCommand: false,
+        mixer: {
+          opacity: 100
+        },
+        data: {
+          display: 'program',
+          partialUpdate: true,
+          slots:
+            {
+              '850_dve': {
+                display: 'program',
+                payload: {
+                  type: 'locators',
+                  style: locatorLabels && locatorLabels.length > 0
+                    ? JSON.parse(splitScreenConfiguration.graphicsTemplateJson)
+                    : '',
+                  ...locatorLabels?.map((label): string => label)
```

Comment:
What is the purpose of this map?

Reply:
Leftover from a previous implementation, but no longer needed. It's been updated.